src/main/java/com/searchclient/clientwrapper/service/SolrDocumentService.java

```diff
@@ -61,10 +74,7 @@ public SolrResponseDTO addDocuments(String collectionName, String payload, boole
 				log.debug(e1.toString());
 
 				String message = "Invalid input JSON array of document.";
```

Comment:
Suggestion 1: No need to assign this to message variable when we have directly pass it to one method i.e, extracted(...) here